src/View/Helper/FormHelper.php

```diff
@@ -1024,6 +1024,10 @@ public function input($fieldName, array $options = [])
         $options = $this->_parseOptions($fieldName, $options);
         $options += ['id' => $this->_domId($fieldName)];
 
+        if (strtolower($options['type']) === 'input') {
+                throw new RuntimeException(sprintf('Invalid type "input" used for field "%s"', $fieldName));
+        }
```

Comment:
Yes it could, do you want me to move it? The end result is more processed lines to get to the exception thought.